src/components/ha-hls-player.ts

```diff
@@ -60,6 +60,10 @@ class HaHLSPlayer extends LitElement {
   private static streamCount = 0;
 
   private _handleVisibilityChange = () => {
+    if (document.pictureInPictureElement !== null) {
```

Comment:
This assumes browser support but it is not guaranteed and `document.pictureInPictureElement` might be `undefined`. I suggest a more defensive check
```suggestion
    if (document.pictureInPictureElement) {
```